system/views/login_page.py

```diff
@@ -0,0 +1,31 @@
+import tkinter as tk
+from tkinter import ttk
+import registration_page
+import user_page
```

Comment:
These imports will give you an error because it needs to take the file structure into account. It should be formatted like this:
from system.views.filename import ClassName